integreat_cms/cms/forms/push_notifications/push_notification_form.py

```diff
@@ -124,7 +138,14 @@ def clean(self):
 
     class Meta:
         model = PushNotification
-        fields = ["channel", "regions", "mode", "scheduled_send_date"]
+        fields = [
+            "channel",
+            "regions",
+            "mode",
+            "scheduled_send_date",
+            "is_template",
+            "template_name",
```

Comment:
I think these are not required, since their corresponding fields are already manually defined above

Reply:
Ah yeah, but it would make more sense to remove the manually defined fields than this ones. The label should be the verbose name of the model field.